Framework/Algorithms/src/MultipleScatteringCorrection.cpp

```diff
@@ -711,21 +716,25 @@ void MultipleScatteringCorrection::calculateL12s(const MultipleScatteringCorrect
       const V3D unitVector = getDirection(posFrom, posTo);
 
       // combined
-      track.reset(posFrom, unitVector);
       track.clearIntersectionResults();
-      const auto rayLen1_container = getDistanceInsideObject(shapeContainer, track);
       track.reset(posFrom, unitVector);
+      // track.clearIntersectionResults();
```

Comment:
Minor point - but please can you remove these commented out lines?

Reply:
done, could you take another look? Thanks